### PR TITLE
Feature/update cask dropdown text combo module

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: coopr-ngui/Gulpfile.js [hunk 1]
```diff
@@ -168,7 +168,11 @@ gulp.task('tpl', function() {
         module: pkg.name + '.features',
         base: __dirname + '/app',
         root: '/assets/'
-      }))
+      })),
+
+      gulp.src(mainBowerFiles({
+      filter: /cask\-angular\-[^\/]+\/tpl\.html\.js/
+    }))
```
Comment (gaarf) [inline coopr-ngui/Gulpfile.js]: indentation is off, and please add a `$` token at the end of the regex.

File: coopr-ngui/bower.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "angular-moment": "0.8.2",
 
     "cask-angular-capitalize": "https://github.com/caskdata/ng-capsules/blob/develop/zip/cask-angular-capitalize.zip?raw=true",
-    "cask-angular-theme": "https://github.com/caskdata/ng-capsules/blob/develop/zip/cask-angular-theme.zip?raw=true"
+    "cask-angular-theme": "https://github.com/caskdata/ng-capsules/blob/develop/zip/cask-angular-theme.zip?raw=true",
+    "cask-angular-dropdown-text-combo": "https://github.com/caskdata/ng-capsules/blob/feature/dropdown-text-combo-update/zip/cask-angular-dropdown-text-combo.zip?raw=true"
```
Comment (gaarf) [inline coopr-ngui/bower.json]: once dependent PR is merge, this should point to `develop` branch, right?

Comment (ajainarayanan) [inline coopr-ngui/bower.json]: Yes that is true.

Comment (ajainarayanan) [inline coopr-ngui/bower.json]: Just updated this now so that when we merge the ng-capsules PR this can be automatically be merged.

File: coopr-ngui/Gulpfile.js (hunk 1 above)
Comment (ajainarayanan) [inline coopr-ngui/Gulpfile.js]: Done.

**Comment by gaarf** [comment]
:+1:  LGTM pending travis :8ball:

File: coopr-ngui/Gulpfile.js [hunk 3]
```diff
@@ -168,7 +168,11 @@ gulp.task('tpl', function() {
         module: pkg.name + '.features',
         base: __dirname + '/app',
         root: '/assets/'
-      }))
+      })),
+
+    gulp.src(mainBowerFiles({
+      filter: /cask\-angular\-[^\/]+\/tpl\.html\.js$/
+    }))
```
Comment (gaarf) [inline coopr-ngui/Gulpfile.js]: wait why is this needed? the regex on line 106 should be enough, no?

**Comment by gaarf** [comment]
forget travis, just merge it, it's fine. :+1:
